crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -126,10 +81,10 @@ fn vertex(vertex_no_morph: Vertex) -> VertexOutput {
         get_instance_index(vertex_no_morph.instance_index)
     );
 #endif // VERTEX_TANGENTS
-#endif // NORMAL_PREPASS
+#endif // NORMAL_PREPASS_OR_DEFERRED_PREPASS
 
-#ifdef MOTION_VECTOR_PREPASS
     out.world_position = bevy_pbr::mesh_functions::mesh_position_local_to_world(model, vec4<f32>(vertex.position, 1.0));
+#ifdef MOTION_VECTOR_PREPASS
```

Comment:
Subtle change here. How come the world_position was not being output before but is now? Because previously it was only needed for calculating motion vectors, and now it's written to the G-Buffer? Seems like It should perhaps not be output for non-deferred? Maybe not a big deal?

Reply:
Yeah deferred needs it. I guess I could make a `MOTION_VECTOR_PREPASS_OR_DEFERRED_PREPASS` like I have with `NORMAL_PREPASS_OR_DEFERRED_PREPASS`?

Reply:
I think that is better so it's not doing unnecessary work in other cases.